bigquery_etl/docs/generate_docs.py

```diff
@@ -18,8 +18,8 @@
 DOCS_DIR = "docs/"
 INDEX_MD = "index.md"
 SQL_REF_RE = r"@sql\((.+)\)"
-SOURCE_URL = "https://github.com/mozilla/bigquery-etl/blob/master"
-EDIT_URL = "https://github.com/mozilla/bigquery-etl/edit/master/"
+SOURCE_URL = "https://github.com/mozilla/bigquery-etl/blob/generated-sql"
+EDIT_URL = "https://github.com/mozilla/bigquery-etl/edit/generated-sql"
```

Comment:
Looking at this, I now realize it's more complicated than I thought 😭 

The "Edit" URL should definitely point at `master` rather than `generated-sql` since the content of `generated-sql` is machine-generated. But some of the views are don't exist in `master`. So what do we do?

I think for now, we should go ahead with this PR as-is, since it will clearly be a better state than what we currently have. But then we'll need to think about a longer-term solution.

Maybe we need to have metadata to track whether a view is automatically generated or not, and treat them differently. At least we can keep this in mind for implementing things in the Glean dictionary.

Reply:
This is indeed more complicated than I thought. Thank you for your patience in reviewing my many attempts at this.
> Maybe we need to have metadata to track whether a view is automatically generated or not, and treat them differently.
👍